{"stack":"SequelizeAccessDeniedError: Access denied for user 'root'@'localhost' (using password: YES)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\connection-manager.js:125:19\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:725:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => Access denied for user 'root'@'localhost' (using password: YES)","timestamp":"2023-11-21T17:21:17.942Z"}
{"stack":"SequelizeConnectionError: Unknown database 'typescript_in_nodejs_starter_db'\n    at C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\connection-manager.js:133:19\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:725:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => Unknown database 'typescript_in_nodejs_starter_db'","timestamp":"2023-11-21T17:26:27.292Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-21T17:57:26.035Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-11-21T17:58:39.549Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-21T18:02:23.817Z"}
{"stack":"SequelizeEagerLoadingError: mytable is not associated to mytable!\n    at Function._getIncludedAssociation (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:715:13)\n    at Function._validateIncludedElement (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:619:53)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:516:37\n    at Array.map (<anonymous>)\n    at Function._validateIncludedElements (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:511:39)\n    at Function.aggregate (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:1956:12)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:2045:19\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"mytable is not associated to mytable!","timestamp":"2023-11-22T18:03:52.705Z"}
{"stack":"SequelizeEagerLoadingError: mytable is not associated to mytable!\n    at Function._getIncludedAssociation (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:715:13)\n    at Function._validateIncludedElement (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:619:53)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:516:37\n    at Array.map (<anonymous>)\n    at Function._validateIncludedElements (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:511:39)\n    at Function.aggregate (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:1956:12)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:2045:19\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"mytable is not associated to mytable!","timestamp":"2023-11-22T18:03:58.339Z"}
{"stack":"SequelizeEagerLoadingError: mytable is not associated to mytable!\n    at Function._getIncludedAssociation (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:715:13)\n    at Function._validateIncludedElement (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:619:53)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:516:37\n    at Array.map (<anonymous>)\n    at Function._validateIncludedElements (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:511:39)\n    at Function.aggregate (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:1956:12)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:2045:19\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"mytable is not associated to mytable!","timestamp":"2023-11-22T18:06:16.422Z"}
{"stack":"SequelizeEagerLoadingError: mytable is not associated to mytable!\n    at Function._getIncludedAssociation (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:715:13)\n    at Function._validateIncludedElement (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:619:53)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:516:37\n    at Array.map (<anonymous>)\n    at Function._validateIncludedElements (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:511:39)\n    at Function.aggregate (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:1956:12)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:2045:19\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"mytable is not associated to mytable!","timestamp":"2023-11-22T18:07:57.447Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'prices' in 'field list'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => Unknown column 'prices' in 'field list'","timestamp":"2023-11-22T18:07:57.518Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'prices' in 'field list'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'prices' in 'field list'","timestamp":"2023-11-22T18:08:18.724Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'prices' in 'field list'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'prices' in 'field list'","timestamp":"2023-11-22T18:10:40.739Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'title' in 'field list'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'title' in 'field list'","timestamp":"2023-11-22T18:11:04.297Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'title' in 'field list'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'title' in 'field list'","timestamp":"2023-11-22T18:11:31.134Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'title' in 'field list'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'title' in 'field list'","timestamp":"2023-11-22T18:11:32.042Z"}
{"stack":"SequelizeEagerLoadingError: mytable is not associated to mytable!\n    at Function._getIncludedAssociation (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:715:13)\n    at Function._validateIncludedElement (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:619:53)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:516:37\n    at Array.map (<anonymous>)\n    at Function._validateIncludedElements (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:511:39)\n    at Function.aggregate (C:\\harvest\\node_modules\\sequelize\\lib\\model.js:1956:12)\n    at C:\\harvest\\node_modules\\sequelize\\lib\\model.js:2045:19\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues [as _onImmediate] (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"mytable is not associated to mytable!","timestamp":"2023-11-22T18:12:07.035Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-23T07:05:05.459Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-23T07:06:13.102Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-23T07:06:39.226Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-23T07:06:55.804Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-23T07:13:00.045Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-23T07:13:00.480Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:170:12)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:267:15)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:64:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-11-25T15:51:39.400Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:170:12)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:267:15)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:64:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-11-25T15:52:04.845Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:170:12)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:267:15)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:64:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-11-25T15:55:40.010Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:64:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-11-25T15:55:41.253Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:64:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-11-25T15:55:41.441Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:170:12)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:267:15)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:64:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-11-25T15:56:09.272Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:64:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-11-25T16:31:45.953Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-25T17:51:33.204Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-25T18:47:11.904Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-25T18:48:40.678Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-25T18:55:17.489Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-26T17:09:06.796Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-27T06:03:07.606Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-11-28T17:41:10.767Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-28T17:42:24.383Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-11-28T17:42:43.937Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-11-28T17:42:46.318Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-11-28T17:42:46.582Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-11-28T17:43:16.621Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-11-28T17:43:20.016Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-11-28T17:44:07.927Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-11-28T17:44:09.124Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-11-29T16:53:21.062Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-12-04T07:05:46.237Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:170:12)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:267:15)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:94:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-12-04T16:36:30.999Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-12-05T09:02:27.659Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-12-06T07:08:10.477Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-12-11T08:49:17.673Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:170:12)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:267:15)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:100:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-12-11T08:52:32.015Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-12-11T10:20:53.354Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-12-11T10:22:15.264Z"}
{"stack":"SequelizeDatabaseError: Unknown column 'NaN' in 'where clause'\n    at Query.formatError (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\harvest\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\harvest\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\harvest\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\harvest\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (node:events:513:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Socket.Readable.push (node:internal/streams/readable:234:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Unknown column 'NaN' in 'where clause'","timestamp":"2023-12-11T10:26:02.971Z"}
{"stack":"Error: Cannot set headers after they are sent to the client\n    at new NodeError (node:internal/errors:399:5)\n    at ServerResponse.setHeader (node:_http_outgoing:645:11)\n    at ServerResponse.header (C:\\harvest\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (C:\\harvest\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (C:\\harvest\\node_modules\\express\\lib\\response.js:158:21)\n    at Object.Ok (C:\\harvest\\src\\app\\shared\\utils\\http-responses.ts:10:26)\n    at C:\\harvest\\src\\app\\controllers\\products.routes.ts:100:7","level":"error","message":"Cannot set headers after they are sent to the client","timestamp":"2023-12-11T10:28:33.421Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-12-12T07:56:35.755Z"}
{"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1740:16)\n    at listenInCluster (node:net:1788:12)\n    at Server.listen (node:net:1876:7)\n    at Function.listen (C:\\harvest\\node_modules\\express\\lib\\application.js:618:24)\n    at C:\\harvest\\src\\app\\startup\\server.ts:94:9\n    at tryCatcher (C:\\harvest\\node_modules\\bluebird\\js\\release\\util.js:16:23)\n    at Promise._settlePromiseFromHandler (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:547:31)\n    at Promise._settlePromise (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:604:18)\n    at Promise._settlePromise0 (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:649:10)\n    at Promise._settlePromises (C:\\harvest\\node_modules\\bluebird\\js\\release\\promise.js:729:18)\n    at _drainQueueStep (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:93:12)\n    at _drainQueue (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:86:9)\n    at Async._drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:102:5)\n    at Immediate.Async.drainQueues (C:\\harvest\\node_modules\\bluebird\\js\\release\\async.js:15:14)\n    at processImmediate (node:internal/timers:476:21)","level":"error","message":"GOT AN UNCAUGHT EXCEPTION => listen EADDRINUSE: address already in use :::3000","timestamp":"2023-12-12T08:14:21.016Z"}
